### ISSUE TITLE
Add example files for easier verification

### ISSUE THREAD

**Issue by dhaeb** [opencore/dsfinvk-java#22]
Can you provide some test index.xml / csv files for clearification? We are able to use the project, but still it was quite a lot to get into it.

**Comment by lfrancke** [comment]
I totally understand and I'd love to. Unfortunately I don't have any examples I'm allowed to share and currently have no time to create some. You're not the first to ask though and I'd happily accept contributions.

I've also already contacted the Bundesfinanzministerium to clarify their licensing of their documents but am not receiving any replies.

I'm sorry. I'll leave this issue open because it's a valid request but I can't help at the moment.

**Comment by dhaeb** [comment]
@lfrancke well we can provide some dummy CSVs and idea.xml if this would be enough? Still, would be great to get some more support from "official" site, as all cash system manufactures have the issue to provide the same kind of data. So if the BMF wants easier results processing, they should be more eager to provide help in form of examples. I know the DSFinV-K 2.2 document quite well, but the bundle contains unfortunately only an idea.xml / DTD. Frankly, this is already quite good, but still some example CSVs would accelerate the dev process. 
Thanks for the answer! I'm going to provide a PR for some changes I need to make to bring this mvn project to compile (related to some Jigsaw-Java madness).

Best!

**Comment by lfrancke** [comment]
Definitely, if you have something it'd be awesome to get some examples in!

For me it's been almost a year now since I looked at all of this in detail but the communication with those guys was painful and Audicon was no help at all.

I'd love your help on this, thank you!
